### PR TITLE
README.md: document port-forward and ingress

### LINKED ISSUE #18: Expose demo/podinfo via the ALB Ingress controller
`eks-quickstart-app-dev` currently creates an ALB Ingress controller, but we do not do anything with it, and it isn't trivial for the end-user how to leverage it. We should add an `Ingress` resource for at least `demo/podinfo` -- given this is our "demo" app of choice -- in order to (1) prove that ingress works, and (2) show end-users how to leverage the ALB in their cluster, to expose services.

> @marccarre podinfo can be safely exposed outside the cluster to demo how the ingress works. The podinfo chart comes with ingress, I would enable it in the HelmRelease file.
>
> Here is an example https://github.com/stefanprodan/gitops-helm-workshop/blob/master/cluster/releases/podinfo.yaml#L23
> 
> _Originally posted by @stefanprodan_

### PR DESCRIPTION
**Pull request by marccarre**
Fixes #18.
Preview available here: https://github.com/weaveworks/eks-quickstart-app-dev/blob/issues/18-how-to-expose-services/README.md#how-to-access-workloads

Special kudos to @stefanprodan who helped debug & fix ingress-related issues for `podinfo`! 🙇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,110 @@ Therefore, depending on your use-case, you may want to:
...
+
+### Port-forwarding
+
+You could port-forward into a pod, so that you (and _only_ you) could access it locally.
+
+For example, for `demo/podinfo`:
+
+- run:
+    ```console
+    kubectl port-forward --namespace=demo podinfo-xxxxxxxxxx-xxxxx 9898
```
Comment (stefanprodan) [inline README.md]:
```suggestion
    kubectl -n demo port-forward svc/podinfo 9898:9898
```

File: README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ For example, for `demo/podinfo`:
 
 - run:
     ```console
-    kubectl port-forward --namespace=demo podinfo-xxxxxxxxxx-xxxxx 9898
+    kubectl -n demo port-forward svc/podinfo 9898:9898
```
Comment (martina-if) [inline README.md]: I think it's better for documentation to use the long version of flags (even if they are well known) so a new kubernetes user doesn't have to look it up before running the command to understand what it is doing.

```suggestion
    kubectl --namespace demo port-forward service/podinfo 9898:9898
```

What do you think?

Comment (marccarre) [reply README.md]: I agree, more verbose, but more explicit for new users. 👍

**Review by martina-if — APPROVED** [review]

**Comment by dholbach** [comment]
Thanks for the ping @marccarre - I used port-forwarding `podinfo` as an example in `site/content/gitops-quickstart/01-apply-gitops.md` too, so I think we're good.

**Pull request merged**